libs/domain/picking/offline/sw-feature.ts

```diff
@@ -1,15 +1,9 @@
-import { AppInitializer } from '@spryker-oryx/core';
+import { AppFeature } from '@spryker-oryx/core';
 import { OfflinePickingFeature } from './feature';
-import { SwPushInitializerService } from './services';
 
-export class SwOfflinePickingFeature extends OfflinePickingFeature {
-  plugins = [];
-
-  providers = [
-    ...super.getProviders(),
-    {
-      provide: AppInitializer,
-      useClass: SwPushInitializerService,
```

Comment:
Why did you remove `SwPushInitializerService` from providers? It's not going to initialize anymore the service.

Reply:
This is not my change, it's from source Taras's branch